<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UICAddChampionDialog</class>
 <widget class="QDialog" name="UICAddChampionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>73</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>171</width>
    <height>73</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>171</width>
    <height>73</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Champion</string>
  </property>
  <widget class="QPushButton" name="btn_accept">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>40</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>40</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
